From 0bfdc760d2cfb3e2c3800310018ac561df458337 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Mon, 4 Nov 2019 01:11:03 +0300
Subject: [PATCH 06/12] arm64: dts: renesas: r8a77970: Update RPC device nodes

This updates RPC device nodes for the consolidated RPC driver.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a77970-eagle.dts | 2 +-
 arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts | 2 +-
 arch/arm64/boot/dts/renesas/r8a77970-v3mzf.dts | 2 +-
 arch/arm64/boot/dts/renesas/r8a77970.dtsi      | 4 ++--
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77970-eagle.dts b/arch/arm64/boot/dts/renesas/r8a77970-eagle.dts
index aa4c65a..5a736aa 100644
--- a/arch/arm64/boot/dts/renesas/r8a77970-eagle.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77970-eagle.dts
@@ -421,7 +421,7 @@
 	};
 };
 
-&qspi0 {
+&rpc0 {
 	pinctrl-0 = <&qspi0_pins &qspi1_pins>;
 	pinctrl-names = "default";
 
diff --git a/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts b/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts
index d6deffe..fad6654 100644
--- a/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts
@@ -195,7 +195,7 @@
 	};
 };
 
-&qspi0 {
+&rpc0 {
 	pinctrl-0 = <&qspi0_pins &qspi1_pins>;
 	pinctrl-names = "default";
 
diff --git a/arch/arm64/boot/dts/renesas/r8a77970-v3mzf.dts b/arch/arm64/boot/dts/renesas/r8a77970-v3mzf.dts
index caecf8c..cabbb0f 100644
--- a/arch/arm64/boot/dts/renesas/r8a77970-v3mzf.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77970-v3mzf.dts
@@ -399,7 +399,7 @@
 	};
 };
 
-&qspi0 {
+&rpc0 {
 	pinctrl-0 = <&qspi0_pins &qspi1_pins>;
 	pinctrl-names = "default";
 
diff --git a/arch/arm64/boot/dts/renesas/r8a77970.dtsi b/arch/arm64/boot/dts/renesas/r8a77970.dtsi
index 456eeb6..603efeb 100644
--- a/arch/arm64/boot/dts/renesas/r8a77970.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77970.dtsi
@@ -767,8 +767,8 @@
 			status = "disabled";
 		};
 
-		qspi0: qspi@ee200000 {
-			compatible = "renesas,qspi-rpc-r8a77970";
+		rpc0: qspi@ee200000 {
+			compatible = "renesas,rpc-r8a77970";
 			reg = <0 0xee200000 0 0x1f0>,
 			      <0 0x08000000 0 0x04000000>,
 			      <0 0xee208000 0 0x100>;
-- 
2.7.4

